### PR TITLE
spelling mistake

### PR DESCRIPTION
**Pull request by J-Seaton**
:q

### CODE REVIEW

**Review by JXSeaton — CHANGES_REQUESTED** [review]
suggested some changes. if approved i will approve

**Comment by J-Seaton** [comment]
made changes

File: newFile.md [hunk 1]
```diff
@@ -6,3 +6,11 @@ normal text! add more text
 
 adding more text and chaning things up!
 yet more changes!
+
+<<<<<<< HEAD
```
Comment (JXSeaton) [inline newFile.md:10]:
```suggestion

```

File: newFile.md [hunk 2]
```diff
@@ -6,3 +6,11 @@ normal text! add more text
...
+adding more text! including a seplling mistake!
+
+lets try this again
+=======
```
Comment (JXSeaton) [inline newFile.md:14]:
```suggestion
=======+++=======
```

File: newFile.md [hunk 3]
```diff
@@ -6,3 +6,11 @@ normal text! add more text
...
+adding more text! Including a spelling mistake!
+>>>>>>> fd1a82074e0dc4499047d3f582353a3cb490694c
```
Comment (JXSeaton) [inline newFile.md:16]:
```suggestion

```

**Review by JXSeaton — CHANGES_REQUESTED** [review]

**Comment by J-Seaton** [comment]
made edits

**Review by JXSeaton — APPROVED** [review]

**Pull request merged**